### ISSUE TITLE
Fix PostCSS plugin ignoring .stylelintignore

### ISSUE THREAD

**Issue by joe223** [stylelint/stylelint#4134]
> Clearly describe the bug

This is an exist issue，although it was closed. https://github.com/stylelint/stylelint/issues/2949

For example, in postcss configuration:

```javascript
module.exports = {
	plugins: [
		require('postcss-import')({
			plugins: [
				require('stylelint')({})
			]
		}),
```
stylelint did not resolve ignore file.

> Which version of stylelint are you using?

```"stylelint": "^9.2.0",```

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

Use `stylelint` Programmatically. PostCSS plugin and Gulp plugin.

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No.

> What did you expect to happen?

Stylelint should resolve ignore file.

> What actually happened (e.g. what warnings or errors did you get)?

Ignore configuration in `.stylelintignore` did not works.

**Comment by evilebottnawi** [comment]
Please create minimum reproducible test repo

**Comment by joe223** [comment]
https://github.com/stylelint/stylelint/blob/0fb8b6d561ca59830cb6d66addf588cfe6169ff1/lib/postcssPlugin.js#L101

I think it's because stylelint postcss plugin did not resolve `.stylelintignore` file.

**Comment by joe223** [comment]
This is a reproducible repo https://github.com/joe223/stylelint-ignore-issue

**Comment by hudochenkov** [comment]
> I think it's because stylelint postcss plugin did not resolve .stylelintignore file.

Looks like you're right, @joe223. PR is welcome!

**Comment by joe223** [comment]
@hudochenkov I'll give it a try.

**Comment by joe223** [comment]
Fix PR https://github.com/stylelint/stylelint/pull/4186
